src/applications/dhp-connected-devices/tests/components/ConnectedDevicesContainer.spec.js

```diff
@@ -55,63 +103,19 @@ describe('Connect Devices Container', () => {
   });
 
   it('should render "You have connected all supported devices" when all supported devices are connected', () => {
-    const allDevicesConnectedState = {
-      connectedDevices: [
-        {
-          vendor: 'Fitbit',
-          authUrl: 'path/to/vetsapi/fitbit/connect/method',
-          disconnectUrl: 'placeholder',
-          connected: true,
-        },
-      ],
-    };
-
-    const allConnectedDevicesContainer = renderInReduxProvider(
+    mockApiRequest(twoDevicesConnectedState);
+    const twoDevicesConnectedContainer = renderInReduxProvider(
       <ConnectedDevicesContainer />,
-      { allDevicesConnectedState },
     );
 
     expect(
-      allConnectedDevicesContainer.findByTestId('no-devices-connected-alert'),
-    ).to.exist;
-
-    const oneDeviceConnectedState = {
-      connectedDevices: [
-        {
-          vendor: 'Fitbit',
-          authUrl: 'path/to/vetsapi/fitbit/connect/method',
-          disconnectUrl: 'placeholder',
-          connected: true,
-        },
-        {
-          vendor: 'Fitbit2',
-          authUrl: 'path/to/vetsapi/fitbit/connect/method',
-          disconnectUrl: 'placeholder',
-          connected: false,
-        },
-      ],
-    };
-
-    const oneConnectedDeviceContainer = renderInReduxProvider(
-      <ConnectedDevicesContainer />,
-      { oneDeviceConnectedState },
-    );
-
-    expect(
-      oneConnectedDeviceContainer.findByTestId('all-devices-connected-alert'),
+      twoDevicesConnectedContainer.findByTestId('all-devices-connected-alert'),
     ).to.be.empty;
```

Comment:
🤔 I think this should be "not empty", no?

Reply:
It should be "not empty". Why is this test passing 🤔

Reply:
My guess is that the test is not working as expected. I know we've had some issues in the past with the `async` nature of the test. Happy to pair on it to debug

Reply:
Thanks. Let's find a time to pair

Reply:
It should be ".to.exist". This test makes sure the appropriate message appears when all available devices are connected.